### ISSUE TITLE
a problem

### ISSUE THREAD

**Issue by yangyang32111** [wangning-001/MANet#1]
Hi，thanks for your great work！ When I run the train.py ,I got this error. Can you give me some guidence? 
![image](https://user-images.githubusercontent.com/67679842/146511813-b727e8b1-1ac2-421c-978f-317a827c411a.png)

**Comment by wangning-001** [comment]
Please check the shape of the mask. This error occurred since the mask is 1 channel, while the code needs 3 channels here. You can expand the mask to 3 channels by torch.cat() to solve this problem.

**Comment by shrutiphutke** [comment]
Hi, I am interested in your work!!!
Will you please share the pre-trained weights of your model for all the three datasets for evaluation?
